llarp/service/endpoint.hpp

```diff
@@ -306,6 +306,12 @@ namespace llarp
       bool
       ShouldBuildMore(llarp_time_t now) const override;
 
+      virtual llarp_time_t
+      DefaultPathAlignmentTimeout() const
+      {
+        return 10s;
```

Comment:
I know it's in a function with a descriptive name, but this value is a named constant in `tun.hpp` and probably should be here as well.

Reply:
it's also actually overridden by that file too :^)

Reply:
On that note, probably should take `Default` out of the method name, and then name a constant as the default and return that here.

Reply:
fair